ios/GitPoint.xcodeproj/project.pbxproj

```diff
@@ -71,6 +70,7 @@
 		DC5C35ED1E37AD1800F3F526 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = DC5C35E31E37AD1800F3F526 /* SimpleLineIcons.ttf */; };
 		DC5C35EE1E37AD1800F3F526 /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = DC5C35E41E37AD1800F3F526 /* Zocial.ttf */; };
 		EB0F176BE52B4561B9FF07AB /* libReactNativeConfig.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D7043BDB577E427391ECFBB0 /* libReactNativeConfig.a */; };
+		CBC20ECDD7FF46A49DF646D6 /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 20A722EC74A74B50BD5DCCC9 /* Feather.ttf */; };
```

Comment:
Have noticed this locally when linking (I'm assuming RNE has a new icon font called `Feather`?) Not sure why I just didn't push it to master but perfectly okay to have it in this PR